src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,73 @@
+package study;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class StringTest {
+
+	@Test
+	@DisplayName("1,2를 ,로 split 했을 때 1과 2로 분리된다")
+	void splitWithTwoElements() {
+	    // Arrange
+	    String input = "1,2";
+
+	    // Act
+	    String[] results = input.split(",");
+
+	    // Assert
+	    assertThat(results).containsExactly("1", "2");
```

Comment:
```suggestion
        assertThat("1,2".split(",")).containsExactly("1", "2");
```
위와 같이 한 라인으로 구현하는 것은 어떻게 생각하나?